main/apr-util/APKBUILD

```diff
@@ -30,16 +30,21 @@ build() {
 		--with-crypto \
 		--with-openssl \
 		--without-sqlite2 \
-		--without-gdbm \
-		|| return 1
-	make || return 1
+		--without-gdbm
+	make
+}
+
+check() {
+	cd "$builddir"
+	# testxlate fails because UTF-7 is unsupported
```

Comment:
This is not very good. Can you somehow easily disable/remove just the failing tests?